### ISSUE TITLE
7$: Improve error when using => on object literal method

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#292]
quick-lint-js reports a crappy message for the following code:
```js
let o = {
  myMethod() => {
  },
};
```
quick-lint-js should instead say something like "methods should not have =>".

This might also apply to class methods.

**Comment by oromate** [comment]
I claim this for-hire task. I expect payment after I complete this task. I will email the quick-lint-js team if I am assigned this task.
